src/main/kotlin/org/rust/cargo/runconfig/command/CargoCommandConfiguration.kt

```diff
@@ -129,8 +129,13 @@ open class CargoCommandConfiguration(
 
     @Throws(RuntimeConfigurationException::class)
     override fun checkConfiguration() {
-        if (isRedirectInput && redirectInputPath?.toPath()?.exists() != true) {
-            throw RuntimeConfigurationWarning("Input file doesn't exist")
+        if (isRedirectInput) {
+            val path = redirectInputPath?.toPathOrNull()
+            when {
+                path?.exists() != true -> throw RuntimeConfigurationWarning("Input file doesn't exist")
```

Comment:
It seems it works incorrectly if I pass `"` on Windows because in this case `path` will be null and we show `Input file doesn't exist` instead of `Input file is not valid`.
`path` can be null only in two cases: `redirectInputPath` is `null` or `redirectInputPath` cannot be converted to valid path on particular OS. And looks like we should show `Input file is not valid` only in the second case

Reply:
Decided that file with invalid sumbols cannot exist on filesystem as well so `Input file doesn't exist` is valid here as well